cli/cmd/vuln_container.go

```diff
@@ -104,7 +104,7 @@ func init() {
 
 	vulContainerShowAssessmentCmd.Flags().BoolVar(
 		&vulCmdState.ImageID, "image_id", false,
-		"tread the provided sha256 hash as image id",
+		"(DEPRECATED) by default we look up both, image_id and image_digest",
```

Comment:
Cobra command has a `MarkDeprecated` function for flags. Can we mark use of this?
github.com/spf13/pflag/flag.go:404

Reply:
Will do.